Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface - Authors fieldtype
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I fill in "Author:" subform with:
+            | Name: | Email: |
```

Comment:
Is it worth using this syntax if we only fill one field ? It would be more readable as one sentence, Like `I fill in the "Author" subform's "name" with an empty string`